test/monster_test/monster_test.c

```diff
@@ -5,6 +5,7 @@
 
 #include "flatcc/support/hexdump.h"
 #include "flatcc/support/elapsed.h"
+#include "../../config/config.h"
```

Comment:
See if #include "config.h" works, config should  be in path. There are a few places with ../../ but it is best avoided.